lib/fpm/package/tar.rb

```diff
@@ -49,7 +49,7 @@ def input(input_path)
   def output(output_path)
     output_check(output_path)
     # Unpack the tarball to the staging path
-    args = ["-cf", output_path, "-C", staging_path]
+    args = ["--owner=root", "--group=root", "-cf", output_path, "-C", staging_path]
```

Comment:
I'm sorry, this line does not belong in this pull request. It has nothing to do with the 'group' field in pacman packages, and was in fact a hack i used to automatically change the owner of files for a .tar.gz "non-package"  in fpm.